test/test_inverse.jl

```diff
@@ -80,7 +80,9 @@ InverseFunctions.inverse(f::Bar) = Bar(inv(f.A))
     @test_throws DomainError inverse(Base.Fix2(^, 0.5))(-5)
     @test_throws DomainError inverse(Base.Fix2(^, 0.51))(complex(-5))
     InverseFunctions.test_inverse(Base.Fix2(^, 0.5), complex(-5))
-    @test_throws DomainError inverse(Base.Fix2(^, 2))(-5)
+    @test_throws DomainError inverse(Base.Fix2(^, 2))
+    @test_throws DomainError inverse(Base.Fix2(^, -4))
```

Comment:
Are the errors for Base.Fix1(^ and log tested somewhere?

Reply:
@aplavin I think there's not test for that yet, correct?